sisl/viz/plotly/gui/server.py

```diff
@@ -0,0 +1,44 @@
+from pathlib import Path
+import webbrowser
+import logging
+import sys
+
+from flask import Flask, send_from_directory
+
+__all__ = ["SERVER_HOST", "SERVER_PORT", "SERVER_ADRESS"]
+
+script_dir = Path(__file__).resolve().parent
+
+app = Flask("SISL GUI", static_folder=str(script_dir / "build"))
+
+# Serve React App
+@app.route("/", defaults={"path": ""})
+@app.route("/<path:path>")
+def serve(path):
+    if path != "" and (Path(app.static_folder) / path).exists():
+        return send_from_directory(app.static_folder, path)
+    else:
+        return send_from_directory(app.static_folder, "index.html")
+
+# Disable all flask logging
+app.logger.disabled = True
+log = logging.getLogger("werkzeug")
+log.disabled = True
+cli = sys.modules["flask.cli"]
+cli.show_server_banner = lambda *x: None
+
+SERVER_HOST = "localhost"
+SERVER_PORT = 7001
```

Comment:
env-var? I.e. users who have 7001 blocked will not be able to change this easily.

Reply:
They can do:

```python
gui.launch(server_kwargs={"port": 8000})
```

from now on. But yes, the default one could probably be an environment variable, I will change it when you make the final way of handling environment variables.

Reply:
Yes, but it would be nice if users running scripts to launch servers could do this on the cmd (unless they explicitly put in the port)

```bash
SISL_FLASK_PORT=8999 python3 run_server.py
```

Reply:
This could be an argument of the `sgui` command isn't it? (This is how you are supposed to run the server from the command line).

```
sgui --server_port 8999
```

But yeah it could also be an environment variable.

PS: The name of the command ("sgui") should probably change as it interferes with the one from Jonas.

Reply:
Ok, I don't really have experience enough with this to say for sure. But ok, lets keep it as is and move forward.

As for the cmd. Once Jonas and your branches are merged I think we should change the `sgui` command to do something like this:
```bash
sgui <backend> <options>
```
So one should do `sgui ase` or `sgui plotly`. This would make a common interface with minimal interference and large consistency.